extensions/interactions/SetInput/directives/oppia-interactive-set-input.directive.ts

```diff
@@ -88,6 +91,9 @@ angular.module('oppia').directive('oppiaInteractiveSetInput', [
                   'I18N_INTERACTIONS_SET_INPUT_ADD_ITEM')
               }
             };
+            if (ctrl.buttonText) {
+              ctrl.schema.ui_config.add_element_text = ctrl.buttonText;
```

Comment:
Ah, this is interesting. Could you please file a follow-up issue to find a way to have this text translated?

Reply:
I talked with @DubeySandeep and it seems that there is no way to serve translations dynamically. Two ways of translating content are either by adding explicit translations to the i18n files or by allowing the content creator to add their translations via the translation tab.

Reply:
We'll have to find/invent a way, then -- hence the request to file an issue rather than solve it in this PR.

Reply:
Filed an [issue](https://github.com/oppia/oppia/issues/8705).

Reply:
Hi @iamprayush, thanks for filing that issue! However, it is way too general. Could you please make it much more specific -- i.e. find a way to support translations for creator-entered customization args? "Support dynamic translations" in general is a big and ill-defined ask, and is hard to work on.

In addition, include a list of all interactions and customization args that this would affect.

Thanks!

Reply:
Hey @seanlip! I've updated the [issue](https://github.com/oppia/oppia/issues/8705) and made it more specific. Please let me know if there are any more changes required.
Also, I think we need to add proper labels to that, but I'm not sure which ones. Any suggestions?

Reply:
The new issue looks good, thanks! I added a label to it stating that it needs a design doc.